test/integration/test_webrtc_api.py

```diff
@@ -0,0 +1,260 @@
+"""
+    Bandwidth
+
+    Bandwidth's Communication APIs  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: letstalk@bandwidth.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import os
+import unittest
+from array import array
+from urllib import response
+
+from hamcrest import *
+
+import bandwidth
+from bandwidth.api import participants_api, sessions_api
+from bandwidth.exceptions import NotFoundException, UnauthorizedException
+from bandwidth.model.create_participant_request import CreateParticipantRequest
+from bandwidth.model.create_participant_response import CreateParticipantResponse
+from bandwidth.model.publish_permissions_enum import PublishPermissionsEnum
+from bandwidth.model.device_api_version_enum import DeviceApiVersionEnum
+from bandwidth.model.participant import Participant
+from bandwidth.model.session import Session
+from bandwidth.model.subscriptions import Subscriptions
+from bandwidth.model.participant_subscription import ParticipantSubscription
+
+
+class TestSessionsApi(unittest.TestCase):
+    """SessionsApi unit test stubs"""
+
+    def setUp(self):
+        # API Client
+        configuration = bandwidth.Configuration(
+            username = os.environ.get('BW_USERNAME'),
```

Comment:
For the environment variables, you could merge in the feature branch and it should add in a util file with all of those variables. Then you could just import and use them here without having to do the whole os.environ thing or having to import os at all. This should make these match the other test files for consistency.